source/common/common/block_memory_hash_set.h

```diff
@@ -15,18 +15,18 @@
 namespace Envoy {
 
 /**
- * Initialization parameters for SharedMemoryHashSet. The options are duplicated
+ * Initialization parameters for BlockMemoryHashSet. The options are duplicated
```

Comment:
Can you search this file for "shared" as well as the cc file? I think there are some comments that need to be fixed in a similar way...